src/logger.ts

```diff
@@ -4,8 +4,10 @@ const prettyJson = format.printf(info => {
 	if (typeof info.message === 'object') {
 		info.message = JSON.stringify(info.message, null, 4);
 	}
-
-	return `${info.level} :: ${info.message}`;
+	
+	const dateObject = new Date();
+	const timeOf = dateObject.toLocaleTimeString();
+	return `${info.level} :: ${info.message} @${timeOf}`;
```

Comment:
I tested this and it looks pretty bad, I think you should change it to

```
return `[${timeOf}] ${info.level} :: ${info.message}`;
```

Reply:
Thanks for the suggestion! I've updated it.